docs/cluster-guide.md

```diff
@@ -0,0 +1,108 @@
+# Guide to setting up a 3-node cluster
+
+This quick start guide provides a step-by-step process for setting up a 3-node EventStoreDB cluster, including environment preparation, certificate generation, and cluster configuration. For additional help, use our [setup wizard](https://configurator.eventstore.com/) for customized installation assistance or refer to our video tutorial on [how to get started with EventStoreDB on Windows](https://www.eventstore.com/blog/getting-started-with-eventstoredb-our-how-to-guide), or [Linux](https://developers.eventstore.com/server/v23.10/installation.html#linux). 
+
+
+
+## Preparing the environment
+
+1. Create a folder named 'Cluster'. Within it, create three subfolders: 'Node1', 'Node2', and 'Node3'.
+2. In each node folder, add a configuration file (either .yml or .txt).
+3. Within each node folder, create the following subfolders:
+    - 'Certificates' to store the certificate and private key. 
+    - 'Data' for all data files.
+    - 'Index' for node indexes.
+    - 'Logs' for log files.
+
+
+
+## Generating certificates
+
+1. Create a folder named 'Generate_Certificate'.
+2. Download the latest version of the certificate generator from [EventStore 'es-gencert-cli'-releases](https://github.com/EventStore/es-gencert-cli/releases) and unzip it into the 'Generate_Certificate' folder.
+3. In a terminal, change to the `es-gencert-cli` directory: 
+
+
+Run this command to generate the root certificate and root private key on Linux: 
+`./es-gencert-cli create-ca -out [Generate_Certificate Path]/ca`
+
+On Windows:
+
+`.\es-gencert-cli.exe create-ca -out [Generate_Certificate Path]\ca`
+
+
+
+4. In the 'Generate_Certificate' directory, run the following commands one at a time for each node (change the path and node number as needed) to generate the certificate and private keys for each node. Use your actual DNS names and recommended IP address configurations to align with production practices. For further details, check out our [cluster with DNS guide](https://developers.eventstore.com/server/v23.10/cluster.html#cluster-with-dns).
+
+For example, if the certificate generator version is 1.2.1 and we’re generating the certificate and private keys for 'Node1' on Linux:
+
+`./es-gencert-cli create-node -ca-certificate /path/to/folder/Generate_Certificate/es-gencert-cli_1.2.1_Linux-x86_64/ca/ca.crt -ca-key` 
+`/path/to/folder/Generate_Certificate/es-gencert-cli_1.2.1_Linux-x86_64/ca/ca.key -out`
+`/path/to/folder/Cluster/Node1/certificates -dns-names your.node1.dns.com`
+
+ The command line will create a folder named 'Certificates' containing the certificate and private key for each node in the directory where we created the folder for Node1 in the folder cluster. 
```

Comment:
I do not understand this line. It says it will create a folder for each node in the directory, but shouldn't there be a separate directory for each node (and therefore only one node in each directory)? Also, the phrase `the directory where we created the folder for Node1 in the folder cluster` is confusing. Does this just mean the `/Cluster/Node1` directory created in Preparing the environment?

Reply:
Line 43 should have been removed actually, it should be Each command will automatically generate the security certificate and the private key for each node in their respective certificates file: